scripts/bulk_price_upload/delete-rds-snapshot.sh

```diff
@@ -14,6 +23,10 @@ then
   echo "No snapshot identifier specified, please supply snapshot identifier in the second argument."
   exit
 else
+  if [[ $SNAPSHOT_IDENTIFIER == "rds"* ]] ; then
```

Comment:
Added check to prevent accidental deletion of snapshots take by cloud platform. They are all start with 'rds'.